store/services/usersettings.js

```diff
@@ -0,0 +1,58 @@
+import feathersVuex from 'feathers-vuex'
+
+let servicePlugin = (feathersClient) => {
+  const { service } = feathersVuex(feathersClient, { idField: '_id' })
+  const servicePath = 'usersettings'
+  const servicePlugin = service(servicePath, {
+    namespace: 'feathers-vuex-usersettings',
+    instanceDefaults: {
+      blacklist: []
+    },
+    getters: {
+      isBlacklisted: (state) => (entity) => {
```

Comment:
This seems really complicated and nested, could we get it cleaner?

Reply:
That seems to be the usual way for getters with params. See https://vuex.vuejs.org/guide/getters.html#method-style-access

Reply:
Uhly but okay